src/skills-builder/test/SkillsBuilder.test.jsx

```diff
@@ -149,4 +165,29 @@ describe('skills-builder', () => {
     fireEvent.click(screen.getByLabelText('Remove career interest: Prospector'));
     expect(dispatchMock).toHaveBeenCalledWith(expected);
   });
+
+  it('should render a <JobSillsSelectableBox> for each career interest the learner has submitted', async () => {
+    render(
+      SkillsBuilderWrapperWithContext(
+        {
+          ...contextValue,
+          state: {
+            ...contextValue.state,
+            currentGoal: 'I want to start my career',
+            currentJobTitle: 'Goblin Lackey',
+            careerInterests: ['Prospector'],
+          },
+        },
+      ),
+    );
+    await act(async () => {
+      fireEvent.click(screen.getByRole('button', { name: 'Next Step' }));
+    });
+
+    const chipComponents = document.querySelectorAll('.pgn__chip');
+    expect(chipComponents[0].textContent).toEqual('finding shiny things');
+    expect(chipComponents[1].textContent).toEqual('mining');
```

Comment:
Thanks for improving this test to include checking that the skills/significance is working as expected.

A nit: Next time, I think this would be better if we have a test explicitly for the component the functionality belongs to. Just as we want to beware functions that "do too much", we also want to beware tests that "do too much". It is also hard to tell what our test coverage is for our subcomponents when do it this way. 

This is better than no tests for the functionality at all though.